crates/re_data_store/benches/arrow2.rs

```diff
@@ -111,7 +111,12 @@ fn erased_clone(c: &mut Criterion) {
             .iter()
             .map(|array| array.total_size_bytes())
             .sum::<u64>();
-        assert!(total_size_bytes as usize >= NUM_ROWS * NUM_INSTANCES * std::mem::size_of::<T>());
+        assert!(
+            total_size_bytes as usize >= NUM_ROWS * NUM_INSTANCES * std::mem::size_of::<T>(),
+            "Size calculated to be {} bytes, but should be at least {} bytes",
+            total_size_bytes,
+            NUM_ROWS * NUM_INSTANCES * std::mem::size_of::<T>()
+        );
```

Comment:
`cargo bench -p re_data_store --bench arrow2` gives me:

```
thread 'main' panicked at crates/re_data_store/benches/arrow2.rs:114:9:
Size calculated to be 66400000 bytes, but should be at least 144000000 bytes
```

Reply:
Oof yeah this didn't age well, `size_of` doesn't make sense now that this uses more complicated types with on-heap data... let me see if I can throw something quick...

Reply:
- Fixed in https://github.com/rerun-io/rerun/pull/4691